configtx/orderer.go

```diff
@@ -816,10 +816,7 @@ func addOrdererValues(ordererGroup *cb.ConfigGroup, o Orderer) error {
 	switch o.OrdererType {
 	case orderer.ConsensusTypeSolo:
 	case orderer.ConsensusTypeKafka:
-		err = setValue(ordererGroup, kafkaBrokersValue(o.Kafka.Brokers), AdminsPolicyKey)
-		if err != nil {
-			return err
-		}
+		return fmt.Errorf("kafka is not supported and deprecated as a consensus type")
```

Comment:
"the kafka consensus type is no longer supported"

Please change to this language in all error messages.

Reply:
done